friendly_traceback/token_utils.py

```diff
@@ -34,18 +38,18 @@ class Token:
     to be transformed will be the string attribute.
     """
 
-    def __init__(self, token):
+    def __init__(self, token: _TokenInfo) -> None:
         self.type = token[0]
         self.string = token[1]
         self.start = self.start_row, self.start_col = token[2]
         self.end = self.end_row, self.end_col = token[3]
         self.line = token[4]
 
-    def copy(self):
+    def copy(self) -> "Token":
         """Makes a copy of a given token"""
```

Comment:
Any reason why this returns "Token" and not Token?

Reply:
@aroberge the class `Token` is not defined yet at the point, so referencing it will fail at runtime. We could use postponed evaluation of annotations as specified in PEP 563 (the `from __future__ import  annotations` statement), but IIRC it doesn't work with Python 3.6.